backend/src/middlewares/auth.middleware.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Response } from 'express';
+import { verify } from 'jsonwebtoken';
+import { PrismaClient } from '@prisma/client';
+import { HttpException } from '@exceptions/HttpException';
+import { DataStoredInToken, RequestWithUser } from '@interfaces/auth.interface';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+const SECRET_KEY = process.env.SECRET_KEY;
```

Comment:
So these type of lines will be only inside config.ts file

Reply:
imported from config file